app/assets/javascripts/actions/task.js

```diff
@@ -0,0 +1,46 @@
+import actionTypes from './actionTypes';
+
+export const taskAdd = (task, story) => ({
+  type: actionTypes.ADD_TASK,
+  task,
+  story
+});
+
+export const removeTask = (task, story) => ({
+  type: actionTypes.REMOVE_TASK,
+  task,
+  story
+});
+
+export const updateTask = (task, story) => ({
+  type: actionTypes.UPDATE_TASK,
+  task,
+  story
+});
+
+export const addTask = (projectId, story, newAttributes) => {
+  return (dispatch, getState, { Task }) => {
+    return Task.newTask(projectId, story.id, newAttributes)
+      .then(task => {
+        dispatch(taskAdd(task, story))
+      });
+  };
+};
+
+export const deleteTask = (projectId, story, taskId) => {
+  return (dispatch, getState, { Task }) => {
+    return Task.excludeTask(projectId, story.id, taskId)
+      .then(() => {
+        dispatch(removeTask(taskId, story))
+      });
+  };
+};
+
+export const updateStatusTask = (projectId, story, task, status) => {
```

Comment:
Changing the function above to `updateTaskSuccess` this one could be just `updateTask`, what do you think?